physics/coulombs_law.py

```diff
@@ -32,7 +32,7 @@ def coulombs_law(q1: float, q2: float, radius: float) -> float:
     17975103584.6
     """
     if radius <= 0:
-        raise ValueError("The radius is always a positive non zero integer")
+        raise ValueError("The radius is always a positive non zero number")
```

Comment:
```suggestion
        raise ValueError("The radius is always a positive number")
```
Could you also remove "non zero"? Because "positive" already implies "non zero"